Client/Frontend/AuthenticationManager/SensitiveViewController.swift

```diff
@@ -54,31 +54,22 @@ class SensitiveViewController: UIViewController {
 
 extension SensitiveViewController {
     private func installBlurredOverlay() {
-        if blurredOverlay == nil {
-            if let snapshot = view.screenshot() {
-                let blurredSnapshot = snapshot.applyBlur(withRadius: 10,
-                                                         blurType: BOXFILTER,
-                                                         tintColor: UIColor(white: 1, alpha: 0.3),
-                                                         saturationDeltaFactor: 1.8,
-                                                         maskImage: nil)
-                let blurredOverlay: UIImageView = .build { $0.image = blurredSnapshot }
-                self.blurredOverlay = blurredOverlay
-                view.addSubview(blurredOverlay)
-
-                NSLayoutConstraint.activate([
-                    blurredOverlay.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-                    blurredOverlay.topAnchor.constraint(equalTo: view.topAnchor),
-                    blurredOverlay.trailingAnchor.constraint(equalTo: view.trailingAnchor),
-                    blurredOverlay.bottomAnchor.constraint(equalTo: view.bottomAnchor)
-                ])
-
-                view.layoutIfNeeded()
-            }
-        }
+        guard backgroundBlurView == nil else { return }
+        let blur = UIBlurEffect(style: .systemMaterialDark)
+        let backgroundBlurView = IntensityVisualEffectView(effect: blur, intensity: 0.2)
+        view.addSubview(backgroundBlurView)
+        backgroundBlurView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            backgroundBlurView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            backgroundBlurView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            backgroundBlurView.topAnchor.constraint(equalTo: view.topAnchor),
+            backgroundBlurView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
```

Comment:
Heya. Dropping in to say that adding a blur view on a UIView [already exists in the codebase](https://github.com/mozilla-mobile/firefox-ios/blob/ae073310226d047dfe9b507ef8ee25ee892e2a3e/Client/Extensions/UIView%2BExtension.swift#L31C6-L31C6). Can we leverage that, somehow? Maybe improve that function so it can be used here, without affecting what it's already doing? I haven't take a look a this in-depth, but it seems like it might be worth exploring, so we're not duplicating code.

Reply:
For sure will look into it. Thank you for your input

Reply:
Done please see cac8a90